README.md

```diff
@@ -21,10 +21,37 @@ To run, `pip install vinyldns-python` and then:
 >>> local_client.list_zones()
 ```
 
+## Contributing
+
+**Requirements**
+
+* `python3`
+* `pip`
+* `virtualenv`
+
+To get started, you will want to setup your virtual environment.  Ensure that you have `virtualenv`
```

Comment:
Think it would be better to break this out to ordered tasks for readability:
```
To get started, you will want to setup your virtual environment.

1. Ensure that you have `virtualenv` installed (`>pip install virtualenv`).
1. Then, simply run `./bootstrap.sh` to setup your environment (unless you really want all these dependencies to be installed locally, which we do not recommend).
1. Activate your virtual environment (`>.virtualenv/bin/activate`) and you will be ready to go!
```